### PR TITLE
Workflows Changes in 9.10

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

File: content/refguide/workflows.md [hunk 1]
```diff
@@ -177,6 +162,8 @@ Now when you run this microflow, it will trigger the selected workflow.
 
 ## 4 Workflow Entities in the System Module {#workflow-entities}
 
+{{% todo %}}[Is information in this section still relevant?]{{% /todo %}}
```
Comment (MariaShaposhnikova) [inline content/refguide/workflows.md:165]: Is information up-to-date in this section?

File: content/refguide/workflows.md [hunk 2]
```diff
@@ -188,6 +175,8 @@ You can find the following workflow-related entities in the System module:
 
 ## 5 Workflow Variables
 
+{{% todo %}}[Is information in this section still relevant?]{{% /todo %}}
```
Comment (MariaShaposhnikova) [inline content/refguide/workflows.md:178]: Is information up-to-date in this section?

File: content/refguide/workflow-setting-up-app.md [hunk 3]
```diff
@@ -77,9 +77,10 @@ While Workflow Commons does provide useful documents out-of-the-box, you might h
 
 ## 5 Workflow Best Practices
 
+{{% todo %}}[Is information in this section still relevant?]{{% /todo %}}
```
Comment (MariaShaposhnikova) [inline content/refguide/workflow-setting-up-app.md:80]: Is information up-to-date in this section?

File: content/refguide/workflows.md (hunk 1 above)
Comment (JefHellemansMendix) [reply content/refguide/workflows.md:165]: **WorkflowTaskDefinition** has become **WorkflowUserTaskDefinition**, and no longer includes system activities. **Obsolete** is also called **IsObsolete**.

File: content/refguide/workflows.md (hunk 2 above)
Comment (JefHellemansMendix) [reply content/refguide/workflows.md:178]: Yes, this is correct.

**Review by JefHellemansMendix — CHANGES_REQUESTED** [review]

File: content/studio/workflows.md [hunk 4]
```diff
@@ -7,17 +7,13 @@ tags: ["workflow", "workflows", "Studio"]
 ---
 
 {{% alert type="warning" %}}
-Workflows in Studio are temporarily unavailable for apps with Mendix version 9.6. We introduced workflows as a Beta feature in Mendix 9 and are now incorporating feedback we received from our community to improve workflows.
+Workflows in Studio are temporarily unavailable for apps with Mendix version 9.6 and above. We introduced workflows as a Beta feature in Mendix 9 and are now incorporating feedback we received from our community to improve workflows.
```
[the remark this answers is not in the document]
Comment (MariaShaposhnikova) [reply content/studio/workflows.md:10]: @JefHellemansMendix , do you mean that workflows will be re-enabled for Studio in 9.10?

[the remark this answers is not in the document]
Comment (JefHellemansMendix) [reply content/studio/workflows.md:10]: Yes, although it might not make it on time for release day.

File: content/refguide/workflows.md (hunk 1 above)
Comment (MariaShaposhnikova) [reply content/refguide/workflows.md:165]: Updated.

**Pull request merged**